server/src/main/java/dev/keva/server/core/KevaServer.java

```diff
@@ -15,26 +17,31 @@
 import java.util.concurrent.TimeUnit;
 
 @Slf4j
-public class KevaDB implements Server {
+@Component
+public class KevaServer implements Server {
```

Comment:
dont think this class needs dependency injection,
can just pass the config (which is the only dependency this class should ever need) in
then init IoC inside here, and pass context to NettyChannelInitializer

Reply:
Updated!